### PR TITLE
SW-5346 Added overdue type and parse results from DeliverableService

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/3061?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#3061** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/3061?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#3060** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/3060?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/DeliverableView/DeliverableStatusBadge.tsx [hunk 1]
```diff
@@ -67,6 +67,20 @@ const DeliverableStatusBadge = (props: DeliverableStatusBadgeProps): JSX.Element
...
+          borderColor: theme.palette.TwClrBrdrDanger,
+          labelColor: theme.palette.TwClrTxtDanger,
+          label: strings.OVERDUE,
+        };
+      case 'Completed':
+        return {
+          backgroundColor: theme.palette.TwClrBgInfoTertiary,
+          borderColor: theme.palette.TwClrBrdrInfo,
+          labelColor: theme.palette.TwClrTxtInfo,
+          label: isAcceleratorRoute ? strings.NEEDS_TRANSLATION : strings.COMPLETED,
```
Comment (constanzauanini) [inline src/components/DeliverableView/DeliverableStatusBadge.tsx:82]: I think this label should always be "Completed"

File: src/components/DeliverablesTable/index.tsx [hunk 2]
```diff
@@ -220,8 +220,9 @@ const DeliverablesTable = ({
 
   useEffect(() => {
     // TODO do something if the request has an error
-    if (deliverablesSearchRequest && deliverablesSearchRequest.data?.deliverables) {
-      setDeliverables(deliverablesSearchRequest.data.deliverables);
+    console.log(deliverablesSearchRequest);
```
Comment (constanzauanini) [inline src/components/DeliverablesTable/index.tsx:223]: remove console.log

**Review by constanzauanini — APPROVED** [review]
Just one comment but looks good!

**Comment by tommylau523** [comment]
### Merge activity

* **Aug 22, 3:38 PM EDT**: @tommylau523 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/3061).

**Pull request merged**